ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java

```diff
@@ -158,8 +158,14 @@ public SelenideElement getInputBySelector(String selector) {
 	public void fillInput(String inputId, String value) {
 		SelenideElement input = this.getElementById(inputId);
 		assertThat(input.getTagName(), is("input"));
-		input.shouldBe(visible).clear();
-		input.shouldBe(visible).sendKeys(value);
+		if (input.getAttribute("type").equals("checkbox")) {
+			if (Boolean.parseBoolean(value)  != input.isSelected()) {
```

Comment:
What about just `input.setSelected(Boolean.valueOf(value));`

Reply:
Yep, that'll work just fine, thanks.